### PR TITLE
Optionally add a cloudwatch readonly user

### PR DESCRIPTION
**Pull request by gertjanmaas**

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -7,3 +7,13 @@ output "trail_arn" {
   description = "Cloud trail arn."
   value       = "${element(concat(aws_cloudtrail.cloudtrail.*.arn, list("")), 0)}"
 }
+
```
Comment (npalm) [inline outputs.tf]: Please add the condition for creating the user as weel. Can you also output the user ARN

**Comment by npalm** [comment]
Please update the README and CHANGELOG as well

File: cloudwatch-readonly-user.tf [hunk 2]
```diff
@@ -0,0 +1,52 @@
+data "aws_iam_policy_document" "cloudwatch_readonly_user_policy_document" {
```
Comment (npalm) [inline cloudwatch-readonly-user.tf]: Not all permissions are related to cloudwatch. maybe cloudwatch user is not the best name. Should we rename it to monitor user?

File: outputs.tf (hunk 1 above)
Comment (gertjanmaas) [reply outputs.tf]: Added the user ARN, but not sure what you mean regarding the condition. Conditions cannot be added to output variables.

File: cloudwatch-readonly-user.tf (hunk 2 above)
Comment (gertjanmaas) [reply cloudwatch-readonly-user.tf]: Renamed

**Pull request merged**